File: problemas/1100/1150.py
x = int(input())
quantidade = 2
soma = 0

while (y := int(input())) <= x:
    pass

i = x + 1
while True:
    soma += i

    if soma + x > y:
        print(quantidade)
        break

    i += 1
    quantidade += 1

File: problemas/1100/1150.dart
import 'dart:io';

void main() {
  final x = int.parse(stdin.readLineSync());
  int quantidade = 2;
  int soma = 0;
  int y;

  while ((y = int.parse(stdin.readLineSync())) <= x) {
  }

  for (int i = x + 1; ; i++, quantidade++) {
    soma += i;

    if (soma + x > y) {
      stdout.writeln(quantidade);
      break;
    }
  }
}